index.html

```diff
@@ -762,9 +762,311 @@
+          <section>
+              <p>UINavigationController</p>
+              <img class="fragment" src="images/nav_controllers_objects.jpg" style="width: 100%; height:100%;">
+              <aside class="notes">
+                  UINavigationController creates and is responsible for the navigation bar and tool bar.  The navigation controller sets itself as the navigation bar's delegate so it can receive messages such as, "back button touched."
+                  The navigation controller manages the navigation stack, which is a "last-in, first-out" collection of custom view controller objects.  The first item added to the stack becomes the <i>root view controller</i> which never gets 'popped' off the stack.
+                  The navigation controller’s primary responsibility is to respond to user actions by pushing new content view controllers onto the stack or popping content view controllers off of the stack.
+                  Each content view controller in the stack configures and pushes the content view controller that is on top of it in the stack.
+                  The navigation controller provides a back button on the navigation bar that pops the topmost view controller automatically when the user taps it.
+              </aside>
+          </section>
+          <section>
+              <p>Creating a navigation interface in code:</p>
+                <pre class="fragment" style="width :1100px"><code>
+- (void)applicationDidFinishLaunching:(UIApplication *)application
+{
+    UIViewController *myViewController = [[MyViewController alloc] init];
+    UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:myViewController];
+    self.window.rootViewController = navigationController;
+    [self.window makeKeyAndVisible];
+}
 
+                </code></pre>
+              <aside class="notes">
+                First line, we allocate and initialize our main view controller.  Then, we set the main view controller as the root view controller (the very first viewcontroller in the navigation stack) while we allocate and initialize our UINavigationController.
+                Next, we take our app's UIWindow, which is provided to us by Xcode when we create our app and set the root view controller as the navigation controller.  Doing this allows the navigation controller access to the window's navigation stack.  Think of the outline of your screen as the window.
+                The UIWindow class defines an object known as a window that manages and coordinates the views an app displays on a device screen. Unless an app can display content on an external device screen, an app has only one window.
+                "makeKeyAndVisible" - This is a convenience method to make the receiver the main window and displays it in front of other windows at the same window level or lower. You can also hide and reveal a window using the inherited hidden property of UIView.
+              </aside>
```

Comment:
I think removing it would be reasonable. You can still talk about how you would do it in code so they appreciate how easy segues are. Totally up to you, don't want to make you rewrite all your slides the day of.